packages/firestore/lite/src/api/snapshot.ts

```diff
@@ -112,8 +113,8 @@ export class QueryDocumentSnapshot<T = firestore.DocumentData>
 export class QuerySnapshot<T = firestore.DocumentData>
   implements firestore.QuerySnapshot<T> {
   constructor(
-    readonly query: firestore.Query<T>,
-    private readonly _docs: Array<QueryDocumentSnapshot<T>>
+    public readonly query: firestore.Query<T>,
```

Comment:
use `readonly` over `public readonly` http://go/ts-style#visibility

Reply:
Done (here and everywhere)